WebApplication/wwwroot/app/admin.js

```diff
@@ -0,0 +1,12 @@
+﻿Vue.component("admin", {
+    data: function () {
+        return {
+            test : "AKO SE OVO PRIKAZE RUTER JE USPEO ADMIN"  
```

Comment:
I assume this is only temporary? You could change it to something more generic 😄